src/main/java/com/kodilla/ecommercee/domain/Cart.java

```diff
@@ -0,0 +1,34 @@
+package com.kodilla.ecommercee.domain;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import javax.persistence.*;
+import javax.validation.constraints.NotNull;
+import java.util.ArrayList;
+import java.util.List;
+
+@NoArgsConstructor
+@AllArgsConstructor
+@Setter
+@Getter
+@Entity(name = "carts")
+public class Cart {
+
+    @Id
+    @GeneratedValue
+    @Column(name = "cart_id", unique = true)
+    private Long id;
+
+    @OneToOne(mappedBy = "cart")
+    @JoinColumn(name = "user_id")
+    private User user;
+
+    @OneToOne(mappedBy = "cart")
+    private Order order;
+
+    @ManyToMany(mappedBy = "cartList")
+    private List<Product> listOfProducts  = new ArrayList<>();
```

Comment:
We need relation to the linking table becouse we’re adding quantity column

Reply:
I have created a CartProducts entity that represents the relationship between the Cart and Product entities. Please re-review.